### ISSUE TITLE
[BUG] Proform Rower Sport RL stroke count always shows 0

### ISSUE THREAD

**Issue by ndesai510** [cagnulein/qdomyos-zwift#1154]
**Describe the bug**
The Stoke count and Stoke length values display 0 for both tiles.  

**To Reproduce**

A debug log was included in a previous bug for the same rower by a different user :
https://github.com/cagnulein/qdomyos-zwift/issues/733

Set virtual rower in options, enable Stroke Count and Stroke Length tiles. 

Both always display 0.

Stroke rate tile displays the correct value. 

Is it possible to check the debug log from the previous debug log to see if the total stroke count  can be added.  If it's not in the above logs, I can provide a new debug log.

**Comment by cagnulein** [comment]
please provide a fresh log, it's always good to check your current setup because different firmware version of the rower could have different results

**Comment by cagnulein** [comment]
@ndesai510 any news?

**Comment by ndesai510** [comment]
@cagnulein - sorry, I decided to just get a Concept 2 rower instead so I do not have the Proform Rower anymore.  The data that the Proform Rower reports is very inaccurate.  Basically reporting double the meters rowed.  Even on a 1 foot short stroke it would say I rowed a full stroke.   It was not an issue on the QZ app side, but from the rower itself.  

Feel free to close this if you needed.  Thanks again for the app.
